crates/cdk-mint-rpc/src/bin/mint_rpc_cli.rs

```diff
@@ -0,0 +1,195 @@
+use std::path::PathBuf;
+
+use cdk_mint_rpc::cdk_mint_client::CdkMintClient;
+use cdk_mint_rpc::mint_rpc_cli::subcommands;
+use cdk_mint_rpc::GetInfoRequest;
+use clap::{Parser, Subcommand};
+use tonic::transport::{Certificate, Channel, ClientTlsConfig, Identity};
+use tonic::Request;
+use tracing::Level;
+use tracing_subscriber::EnvFilter;
+
+const DEFAULT_WORK_DIR: &str = ".cdk-mint-rpc-cli";
+
+#[derive(Parser)]
+#[command(version, about, long_about = None)]
+struct Cli {
+    /// Address of RPC server
+    #[arg(short, long, default_value = "http://127.0.0.1:8086")]
+    addr: String,
+
+    /// Logging level
+    #[arg(short, long, default_value = "debug")]
+    log_level: Level,
+
+    /// Path to working dir
+    #[arg(short, long)]
+    work_dir: Option<PathBuf>,
+
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    /// Get info
+    GetInfo,
+    /// Update motd
+    UpdateMotd(subcommands::UpdateMotdCommand),
+    /// Update short description
+    UpdateShortDescription(subcommands::UpdateShortDescriptionCommand),
+    /// Update long description
+    UpdateLongDescription(subcommands::UpdateLongDescriptionCommand),
+    /// Update name
+    UpdateName(subcommands::UpdateNameCommand),
+    /// Update icon url
+    UpdateIconUrl(subcommands::UpdateIconUrlCommand),
+    /// Add Url
+    AddUrl(subcommands::AddUrlCommand),
+    /// Remove Url
+    RemoveUrl(subcommands::RemoveUrlCommand),
+    /// Add contact
+    AddContact(subcommands::AddContactCommand),
+    /// Remove contact
+    RemoveContact(subcommands::RemoveContactCommand),
+    /// Update nut04
+    UpdateNut04(subcommands::UpdateNut04Command),
+    /// Update nut05
+    UpdateNut05(subcommands::UpdateNut05Command),
+    /// Update quote ttl
+    UpdateQuoteTtl(subcommands::UpdateQuoteTtlCommand),
+    /// Update Nut04 quote
+    UpdateNut04QuoteState(subcommands::UpdateNut04QuoteCommand),
+    /// Rotate next keyset
+    RotateNextKeyset(subcommands::RotateNextKeysetCommand),
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let args: Cli = Cli::parse();
+    let default_filter = args.log_level;
+
+    let sqlx_filter = "sqlx=warn,hyper_util=warn,reqwest=warn";
+
+    let env_filter = EnvFilter::new(format!("{},{}", default_filter, sqlx_filter));
+
+    // Parse input
+    tracing_subscriber::fmt().with_env_filter(env_filter).init();
+
+    let cli = Cli::parse();
+
+    let work_dir = match &args.work_dir {
+        Some(work_dir) => work_dir.clone(),
+        None => {
+            let home_dir = home::home_dir().unwrap();
+
+            home_dir.join(DEFAULT_WORK_DIR)
+        }
+    };
+
+    std::fs::create_dir_all(&work_dir)?;
+    tracing::debug!("Using work dir: {}", work_dir.display());
+
+    let channel = if work_dir.join("tls").is_dir() {
+        // TLS directory exists, configure TLS
```

Comment:
```suggestion
        tracing::info!("Found TLS directory, configuring TLS");
```

A log statement would be more helpful IMO.